package.json

```diff
@@ -32,9 +32,10 @@
     "react-redux": "^8.0.1",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.0",
+    "react-select": "^5.4.0",
```

Comment:
As we already have MUI, can you try customizing `autocomplete` or `select` to achieve the dropdown with avatar image?

Reply:
Sure @koustov, I will have a look.

Reply:
@koustov, I checked the MUI Select component and it does provide support for showing images in options, but it does not support displaying images for selected options.
Something like this.
![image](https://user-images.githubusercontent.com/59080942/183746031-3d464d5c-04e3-4a36-8994-e4d1a834a8d9.png)

Reply:
@Sachin-chaurasiya I just made a quick and dirty one for reference. https://codesandbox.io/s/stoic-ishizaka-3nbwiu

You have to make tons of changes on top of it however it satisfies what we are looking for.

Reply:
Thanks, @koustov, I will work on it.

Reply:
Hey @koustov, I found a better way of doing it, we can use the `renderValue` prop to render the creator with an image and label.

You can checkout the implementation here https://codesandbox.io/s/custom-select-wfy40v?file=/demo.tsx